### PR TITLE
add information how DelimitedFiles can be used

### LINKED ISSUE #2772: Question: Small Delimited file into DataFrame
Hi,
I have read the following:
https://dataframes.juliadata.org/stable/man/importing_and_exporting/
Is there an alternative way to get a small text file delimited with '\t' into a dataframe in the first run without waiting 12 seconds for CSV.File to compile? 
The text file is different everytime.
Thank you

### PR DESCRIPTION
**Pull request by bkamins**
Fixes #2772

### CODE REVIEW

**Review by quinnj — APPROVED** [review]

File: docs/src/man/importing_and_exporting.md
Comment (nalimilan) [inline docs/src/man/importing_and_exporting.md:100]:
```suggestion
  as some of the columns are numeric and some are string.
```

File: docs/src/man/importing_and_exporting.md
Comment (bkamins) [reply docs/src/man/importing_and_exporting.md:33]: OK.

Comment (bkamins) [reply docs/src/man/importing_and_exporting.md]: added

Comment (bkamins) [reply docs/src/man/importing_and_exporting.md]: while waiting for the approval I have added one more tutorial to the list of curated tutorials

Comment (bkamins) [reply docs/src/man/importing_and_exporting.md]: @nalimilan - looks good now?

File: docs/src/man/importing_and_exporting.md
Comment (nalimilan) [inline docs/src/man/importing_and_exporting.md:125]:
```suggestion
* Microsoft Excel (XLSX): [XLSX.jl](https://github.com/felipenoris/XLSX.jl)
```

File: docs/src/man/importing_and_exporting.md
Comment (nalimilan) [inline docs/src/man/importing_and_exporting.md:3]:
```suggestion
## CSV Files
```

File: docs/src/man/importing_and_exporting.md
Comment (nalimilan) [inline docs/src/man/importing_and_exporting.md:119]:
```suggestion
Other data formats are supported for reading and writing in the following packages
(non exhaustive list):
```

File: docs/src/man/importing_and_exporting.md
Comment (nalimilan) [inline docs/src/man/importing_and_exporting.md:120]:
```suggestion
* Apache Arrow (including Feather v2): [Arrow.jl](https://github.com/JuliaData/Arrow.jl)
* Apache Feather (v1): [Feather.jl](https://github.com/JuliaData/Feather.jl)
```

**Review by nalimilan — APPROVED** [review]
Just a few suggestions.

**Comment by bkamins** [comment]
Thank you!

**Pull request merged**
